workspace.json

```diff
@@ -6035,7 +6072,11 @@
           "builder": "@nrwl/workspace:run-commands",
           "options": {
             "outputPath": "apps/services/endorsements/api/src/openapi.yaml",
-            "command": "yarn ts-node -P apps/services/endorsements/api/tsconfig.app.json apps/services/endorsements/api/src/buildOpenApi.ts"
+            "commands": [
+              "nx run services-endorsements-api:build",
+              "yarn ts-node -P apps/services/endorsements/api/tsconfig.app.json dist/apps/services/endorsements/api/buildOpenApi.js"
```

Comment:
shouldn't it be `.ts`?

Reply:
No, we are building the file since it uses the NestJS root app module to generate the schema and when not build it does not have access to the swagger plugin since that is supplied via webpack config applied in build step.
There might be better ways around this such as running `nest-cli` to do this generation, we did not look into this further.
But this should probably not be using `ts-node`, will change to `node`.